src/services/game.service.jsx

```diff
@@ -1,3 +1,18 @@
+import { getCards as getRickAndMortyCards } from "./rickmorty.service.mjs";
+import { getCards as getDisneyCards } from "./disney.services.mjs";
+import { getCards as getAmiiboCards } from "./amiibo.services.mjs";
+
+export const getCards = async function (theme, amount) {
+  if (theme === "disney") {
+    return getDisneyCards(amount);
+  }
+  if (theme === "amiibo") {
+    return getAmiiboCards(amount);
+  }
+
+  return getRickAndMortyCards(amount);
+};
+
```

Comment:
Die getCards könntest du evtl. in ein eigenes modul packen.

Reply:
Ist es ja in game.services ;) nochmal rausziehen bringt keinen Mehrwert, weil das ja eine game spezifische Aufgabe ist oder wohin würdest du die packen wollen?

Reply:
Stimmt, hat erstmal keinen Mehrwert, lass es so.